Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md

```diff
@@ -44,10 +44,32 @@ If you’d like to delete all the default actions so that you can start from an
 * To delete an existing Action, either right-click it and select __Delete__ from the context menu.
 * To duplicate an existing Action, either right-click it and select __Duplicate__ from the context menu.
 
+## Action type and Control type
+
 If you select an Action, you can edit its properties in the right-hand pane of the window:
 
 ![Action Properties](Images/ActionProperties.png)
 
+#### Action Type
+
+The Action Type setting allows to to select between **Button**, **Value** or **PassThrough**.
+
+These options relate to whether this action should represent a discrete on/off button-style interaction or a value that can change over time while the control is being used.
+
+For device controls such as keyboard keys, mouse clicks, or gamepad buttons, select **Button**. For device controls such as mouse movement, a joystick or gamepad stick, or device orientation that provide continuously changing input over a period of time, select **Value**.
+
+The Button and Value types of action also provides data about the action such as whether it has started and stopped, and conflict resolution in situations where multiple bindings are mapped to the same action.
+
+The third option, **PassThrough**, is also a value type, and as such is suitable for the same types of device controls as value. The difference is that actions set to PassThrough only provide basic information about the values incoming from the device controls bound to it, and does not provide the extra data relating to the phase of the action, nor does it perform conflict resolution in the case of multiple controls mapped to the same action.
+
+For more detail about how these types work, see [action types](RespondingToActions.html#action-types) and [default interactions](Interactions.html#default-interaction).
+
+#### Control Type
+
+The Control Type setting allows you to select the type of control expected by the action. This limits the controls shown when setting up bindings in the UI and also limits which contols can be bound interactively to the action.
+
+For example, if you select 2D axis, only those device controls that can supply a 2D vector as value are available as options for the binding control path (see directly below).
```

Comment:
Could we list some common ones here: 
E.g. suggest Vector2 as a common selection for gamepad stick input, or for use with a composite WASD input

I'm still not sure why I would use 2D axis over Vector2 or when I would use dpad

2dvector and dpad are mentioned here - with no clarity over why using one over the other : https://docs.unity3d.com/Packages/com.unity.inputsystem@1.7/manual/ActionBindings.html#2d-vector
Could/should we list other types on this page too?

This page shows dpad but not why its used over 2d-vector
https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/Actions.html#creating-actions-in-code

Reply:
add extra information that Axis or Dpad offers extra processing over 2d vector
Axis adds dead zone and clamping.
Dpad changes the interface to up/down/left/right digital response.